third_party/bno055/RPi_BNO055.cpp

```diff
@@ -348,6 +348,47 @@ imu::Vector<3> Adafruit_BNO055::getVector(adafruit_vector_type_t vector_type) {
   return xyz;
 }
 
+bool Adafruit_BNO055::getVectors( jcc::Vec3& accel, jcc::Vec3& gyro, jcc::Vec3& mag ) {
+  constexpr uint8_t num_registers_to_read = 6 * 3;
```

Comment:
`constexpr` define what 3 and 6 are, what do those numbers mean?

Reply:
This number is 18, so the max index in this array will be 17, but.... [cont'd on page 2]